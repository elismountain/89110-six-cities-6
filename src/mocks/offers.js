export default [{
  bedrooms: 3,
  city: {
    location: {
      latitude: 52.370216,
      longitude: 4.895168,
      zoom: 10
    },
    name: `Amsterdam`
  },
  description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  goods: [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  host: {
    avatarUrl: `img/avatar-angelina.jpg`,
    id: 3,
    isPro: true,
    name: `Angelina`
  },
  id: 1,
  images: [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`
  ],
  isFavorite: false,
  isPremium: false,
  location: {
    latitude: 52.3909553943508,
    longitude: 4.85309666406198,
    zoom: 8
  },
  maxAdults: 4,
  previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
  price: 100,
  rating: 5,
  title: `Beautiful & luxurious studio at great location`,
  type: `apartment`
}, {
  bedrooms: 3,
  city: {
    location: {
      latitude: 52.370216,
      longitude: 4.895168,
      zoom: 10
    },
    name: `Amsterdam`
  },
  description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  goods: [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  host: {
    avatarUrl: `img/avatar-angelina.jpg`,
    id: 3,
    isPro: true,
    name: `Angelina`
  },
  id: 2,
  images: [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
    `https//assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`
  ],
  isFavorite: false,
  isPremium: false,
  location: {
    latitude: 52.369553943508,
    longitude: 4.85309666406198,
    zoom: 8
  },
  maxAdults: 4,
  previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
  price: 120,
  rating: 3.1,
  title: `Beautiful & luxurious studio at great location`,
  type: `apartment`
},

{
  bedrooms: 3,
  city: {
    location: {
      latitude: 52.370216,
      longitude: 4.895168,
      zoom: 10
    },
    name: `Amsterdam`
  },
  description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  goods: [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  host: {
    avatarUrl: `img/avatar-angelina.jpg`,
    id: 3,
    isPro: true,
    name: `Angelina`
  },
  id: 3,
  images: [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`
  ],
  isFavorite: false,
  isPremium: false,
  location: {
    latitude: 52.3909553943508,
    longitude: 4.929309666406198,
    zoom: 8
  },
  maxAdults: 4,
  previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
  price: 130,
  rating: 4,
  title: `Beautiful & luxurious studio at great location`,
  type: `apartment`
}, {
  bedrooms: 3,
  city: {
    location: {
      latitude: 52.370216,
      longitude: 4.895168,
      zoom: 10
    },
    name: `Amsterdam`
  },
  description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  goods: [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  host: {
    avatarUrl: `img/avatar-angelina.jpg`,
    id: 3,
    isPro: true,
    name: `Angelina`
  },
  id: 4,
  images: [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`
  ],
  isFavorite: false,
  isPremium: false,
  location: {
    latitude: 52.3809553943508,
    longitude: 4.939309666406198,
    zoom: 8
  },
  maxAdults: 4,
  previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
  price: 140,
  rating: 2,
  title: `Beautiful & luxurious studio at great location`,
  type: `apartment`
}
];
