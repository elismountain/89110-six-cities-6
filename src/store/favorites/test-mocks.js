export const favOfferRaw = {
  "city": {
    "name": `Paris`,
    "location": {
      "latitude": 48.85661,
      "longitude": 2.351499,
      "zoom": 13
    }
  },
  "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
  "images": [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`
  ],
  "title": `Canal View Prinsengracht`,
  "is_favorite": true,
  "is_premium": false,
  "rating": 3.5,
  "type": `apartment`,
  "bedrooms": 2,
  "max_adults": 7,
  "price": 340,
  "goods": [
    `Baby seat`,
    `Towels`,
    `Dishwasher`,
    `Coffee machine`,
    `Air conditioning`,
    `Washer`,
    `Fridge`,
    `Breakfast`,
    `Laptop friendly workspace`
  ],
  "host": {
    "id": 25,
    "name": `Angelina`,
    "is_pro": true,
    "avatar_url": `img/avatar-angelina.jpg`
  },
  "description": `Peaceful studio in the most wanted area in town. Quiet house Near of everything. Completely renovated. Lovely neighbourhood, lot of trendy shops, restaurants and bars in a walking distance.`,
  "location": {
    "latitude": 48.84461,
    "longitude": 2.374499,
    "zoom": 16
  },
  "id": 1
};

export const favOfferAdapted = {
  "city": {
    "name": `Paris`,
    "location": {
      "latitude": 48.85661,
      "longitude": 2.351499,
      "zoom": 13
    }
  },
  "previewImage": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
  "images": [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`
  ],
  "title": `Canal View Prinsengracht`,
  "isFavorite": true,
  "isPremium": false,
  "rating": 3.5,
  "type": `apartment`,
  "bedrooms": 2,
  "maxAdults": 7,
  "price": 340,
  "goods": [
    `Baby seat`,
    `Towels`,
    `Dishwasher`,
    `Coffee machine`,
    `Air conditioning`,
    `Washer`,
    `Fridge`,
    `Breakfast`,
    `Laptop friendly workspace`
  ],
  "host": {
    "id": 25,
    "name": `Angelina`,
    "isPro": true,
    "avatarUrl": `img/avatar-angelina.jpg`
  },
  "description": `Peaceful studio in the most wanted area in town. Quiet house Near of everything. Completely renovated. Lovely neighbourhood, lot of trendy shops, restaurants and bars in a walking distance.`,
  "location": {
    "latitude": 48.84461,
    "longitude": 2.374499,
    "zoom": 16
  },
  "id": 1
};

export const favOffersRaw = [
  {
    "city": {
      "name": `Paris`,
      "location": {
        "latitude": 48.85661,
        "longitude": 2.351499,
        "zoom": 13
      }
    },
    "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
    "images": [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`
    ],
    "title": `The Joshua Tree House`,
    "is_favorite": true,
    "is_premium": false,
    "rating": 2.1,
    "type": `apartment`,
    "bedrooms": 4,
    "max_adults": 8,
    "price": 170,
    "goods": [
      `Cable TV`,
      `Baby seat`,
      `Coffee machine`,
      `Laptop friendly workspace`,
      `Fridge`,
      `Washer`,
      `Dishwasher`,
      `Washing machine`,
      `Towels`,
      `Air conditioning`,
      `Breakfast`
    ],
    "host": {
      "id": 25,
      "name": `Angelina`,
      "is_pro": true,
      "avatar_url": `img/avatar-angelina.jpg`
    },
    "description": `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
    "location": {
      "latitude": 48.84761,
      "longitude": 2.356499,
      "zoom": 16
    },
    "id": 6
  },
  {
    "city": {
      "name": `Amsterdam`,
      "location": {
        "latitude": 52.37454,
        "longitude": 4.897976,
        "zoom": 13
      }
    },
    "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
    "images": [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`
    ],
    "title": `Canal View Prinsengracht`,
    "is_favorite": true,
    "is_premium": false,
    "rating": 3.4,
    "type": `room`,
    "bedrooms": 1,
    "max_adults": 2,
    "price": 186,
    "goods": [
      `Breakfast`,
      `Laptop friendly workspace`,
      `Washer`,
      `Baby seat`,
      `Towels`,
      `Air conditioning`,
      `Fridge`
    ],
    "host": {
      "id": 25,
      "name": `Angelina`,
      "is_pro": true,
      "avatar_url": `img/avatar-angelina.jpg`
    },
    "description": `Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.`,
    "location": {
      "latitude": 52.37554,
      "longitude": 4.9019759999999994,
      "zoom": 16
    },
    "id": 14
  },
  {
    "city": {
      "name": `Hamburg`,
      "location": {
        "latitude": 53.550341,
        "longitude": 10.000654,
        "zoom": 13
      }
    },
    "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
    "images": [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`
    ],
    "title": `Waterfront with extraordinary view`,
    "is_favorite": true,
    "is_premium": false,
    "rating": 2.5,
    "type": `apartment`,
    "bedrooms": 2,
    "max_adults": 6,
    "price": 174,
    "goods": [
      `Air conditioning`,
      `Breakfast`,
      `Fridge`,
      `Towels`,
      `Laptop friendly workspace`,
      `Washing machine`,
      `Dishwasher`,
      `Coffee machine`,
      `Baby seat`,
      `Washer`
    ],
    "host": {
      "id": 25,
      "name": `Angelina`,
      "is_pro": true,
      "avatar_url": `img/avatar-angelina.jpg`
    },
    "description": `Design interior in most sympathetic area! Complitely renovated, well-equipped, cosy studio in idyllic, over 100 years old wooden house. Calm street, fast connection to center and airport.`,
    "location": {
      "latitude": 53.546341000000005,
      "longitude": 10.022654000000001,
      "zoom": 16
    },
    "id": 51
  }
];
