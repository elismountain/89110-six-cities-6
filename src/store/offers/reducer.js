import {ActionType} from '../action';
import {adaptOfferData, adaptReviewsData} from '../../services/adapter';

const initialState = {
  offer: null,
  reviews: [],
};

const reducer = (state = initialState, action) => {
  switch (action.type) {
    case ActionType.LOAD_OFFER:
      return {
        ...state,
        offer: adaptOfferData(action.payload),
        offerNotFound: false
      };
    case ActionType.LOAD_REVIEWS:
      return {
        ...state,
        reviews: adaptReviewsData(action.payload)
      };
    default:
      return state;
  }
};

export {reducer};
