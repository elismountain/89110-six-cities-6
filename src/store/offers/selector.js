import {NameSpace} from '../root-reducer';

const getByProp = (state) => (prop) => state[NameSpace.OFFER][prop];

export const getCurrentOffer = (state) => getByProp(state)(`offer`);
export const getReviews = (state) => getByProp(state)(`reviews`);
export const getNotFoundStatus = (state) => getByProp(state)(`offerNotFound`);

export const getNearbyOffers = (state) => {
  return state[NameSpace.MAIN].offers.filter((item) => state[NameSpace.OFFER].nearby.includes(item.id));
};
