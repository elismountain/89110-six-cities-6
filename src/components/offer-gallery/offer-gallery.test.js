import React from 'react';
import {render} from '@testing-library/react';
import OfferGallery from '../offer-gallery/offer-gallery';
import {offerAdapted} from '../../store/offers/test-mocks';

it(`OfferGallery should render correctly`, () => {
  const {container} = render(
      <OfferGallery images={offerAdapted.images} />
  );

  expect(container).toMatchSnapshot();
});
