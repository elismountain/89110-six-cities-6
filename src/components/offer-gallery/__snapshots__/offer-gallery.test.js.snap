// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferGallery should render correctly 1`] = `
<div>
  <div
    class="property__gallery-container container"
  >
    <div
      class="property__gallery"
    >
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
